functions/postgre-sql-node/handler.js

```diff
@@ -0,0 +1,53 @@
+const pg = require('pg')
+
+const PG_HOST=process.env.PG_HOST;
+const PG_USER=process.env.PG_USER;
+const PG_DATABASE=process.env.PG_DATABASE;
+const PG_PASSWORD=process.env.PG_PASSWORD;
+const PG_PORT=parseInt(process.env.PG_PORT,10);
+
+const pool = new pg.Pool({
+  user: PG_USER,
+  host: PG_HOST,
+  database: PG_DATABASE,
+  password: PG_PASSWORD,
+  port: PG_PORT
+})
+
+exports.handle = async (event, context, callback) => {
+  try {
+      const { rows } = await query("SELECT * FROM table LIMIT 10")
+      console.log(JSON.stringify(rows[0]))
+      const response = {
+          "statusCode": 200,
+          "headers": {
+              "Content-Type" : "application/json"
+          },
+          "body": JSON.stringify(rows),
+          "isBase64Encoded": false
+      };
+      callback(null, response);
+  } catch (err) {
+      console.log('Database ' + err)
+      callback(null, 'Database ' + err);
```

Comment:
I would pass the err directly:
 https://www.scaleway.com/en/docs/serverless/functions/reference-content/code-examples/#returning-a-function-result-in-node

```suggestion
      callback(err);
```

If you want to add the database prefix you could do:
```js
err.message = `Database: ${err.message}`
callback(err)
```